go.mod

```diff
@@ -131,7 +131,7 @@ require (
 	go.opentelemetry.io/otel/sdk v1.14.0
 	go.opentelemetry.io/otel/trace v1.14.0
 	go.opentelemetry.io/proto/otlp v0.19.0
-	golang.org/x/crypto v0.6.0 // replaced
+	golang.org/x/crypto v0.7.0 // replaced
```

Comment:
FYI @jentfoo @jakule. Another week, another /x/crypto version. I don't see anything urgent, but we might as well update the fork when you have the time.